dangermattic.gemspec

```diff
@@ -50,4 +53,5 @@ Gem::Specification.new do |spec|
   #
   # This will stop test execution and let you inspect the results
   spec.add_development_dependency 'pry'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
We need to disable this and make sure not to add it (or add it and set it to false), otherwise we won't be able to use CI to release the gem automatically for us every time we create a git tag.

See also https://github.com/wordpress-mobile/release-toolkit/pull/475 and p1683817235590059-slack-C02KLTL3MKM